### PR TITLE
DB-10754 ALTER TABLE SET DEFAULT now uses system default 

### PR DESCRIPTION
**Pull request by arnaud-splice**

### CODE REVIEW

**Comment by arnaud-splice** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/2166/

**Review by msirek — APPROVED** [review]

**Comment by arnaud-splice** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Review by dgomezferro — APPROVED** [review]

**Review by dgomezferro — COMMENTED** [review]
@arnaud-splice are we checking on the ITs that the `alter table set default` takes place after running it?

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/actions/AlterTableConstantOperationIT.java [hunk 1]
```diff
@@ -699,13 +694,19 @@ private void testAlterAddColumnWithEmptyDefault(boolean insertBeforeAlter, Strin
...
 
     @Test
     public void testAlterAddColumnInFilledTableWithEmptyDefault() throws Exception {
         String tableName = "test_alter_add_column_in_filled_table_with_empty_default";
-        testAlterAddColumnWithEmptyDefault(true, tableName);
+        testAlterAddColumnWithEmptyDefault(true, tableName, false, false);
+    }
+
+    @Test
+    public void testAlterAddColumnInEmptyTableThenSetAnEmptyDefault() throws Exception {
```
Comment (arnaud-splice) [inline splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/actions/AlterTableConstantOperationIT.java:707]: @dgomezferro this test is doing that:
setDefaultSeparately = true, so we'll create the column, then alter the default
insertBeforeAlter = false, so we'll alter the table, then insert

Comment (dgomezferro) [reply splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/actions/AlterTableConstantOperationIT.java:707]: I see, thanks!

**Review by dgomezferro — APPROVED** [review]

**Review by carolp-503 — APPROVED** [review]
+1

**Pull request merged**